app/scripts/modules/kubernetes/cluster/configure/CommandBuilder.js

```diff
@@ -168,7 +168,8 @@ module.exports = angular.module('spinnaker.kubernetes.clusterCommandBuilder.serv
         return {
           fromTrigger: true,
           repository: trigger.repository,
-          registry: trigger.registry,
+          account: trigger.account,
```

Comment:
Could we keep this for now? It's just used to render where the image is coming from and will break the UI. I'll switch it to account, and when I do I'll remove this `registry:` line. You can drop a TODO with my name in it here.